src/main/java/org/umssdiplo/automationv01/core/managepage/SHLogin.java

```diff
@@ -34,10 +41,15 @@ public void clickCloseLoginButton() {
         CommonEvents.clickButton(closeLoginBtn);
     }
 
+    public void itsVisibleLogOutButton() {
```

Comment:
user this name please: isLogoutButtonDisplayed(), if you are asking something, you need to return a boolean.

if you need to wait, then waitUntilLogoutButtonVisibles()